lib/docker-sync/preconditions.rb

```diff
@@ -43,4 +43,12 @@ def self.unison_available
       raise('Could not find unison binary in path. Please install it, e.g. using "brew install unison"')
     end
   end
+
+  def self.unox_available
+    if (find_executable0 'unison-fsmonitor').nil?
+      raise('Could not find unison-fsmonitor binary in path. Please install it, see https://github.com/hnsl/unox, or simply run :
+      wget -O /usr/local/bin/unison-fsmonitor https://raw.githubusercontent.com/hnsl/unox/master/unox.py \
+      && chmod +x /usr/local/bin/unison-fsmonitor')
+    end
```

Comment:
The reason was, not everybody has brew. In your case, no dependencys are needed and its the only way to install. AFAIK wget is not part of OSX by default, you might want to use curl.

I will change the others also by the way, adding a question, like for the images